lib/api/controller/server.js

```diff
@@ -235,6 +237,28 @@ class ServerController extends NativeController {
     return apiDefinition;
   }
 
+  async swagger (request) {
+    const format = 
+      // The YAML response format is only valid with an HTTP request
+      request.context.connection.protocol === 'http'&& request.input.args.format
```

Comment:
Why can't we have YAML in WebSocket or MQTT? YAML it's just a string so we could put it in the result